tests/jballerina-unit-test/src/test/resources/test-src/execution/proj9/src/c/main.bal

```diff
@@ -0,0 +1,14 @@
+import a;
```

Comment:
Shall we add license header to the test bal files?

Reply:
The bal files in the previous test projects does not contain the license headers. 
[project 1](https://github.com/ballerina-platform/ballerina-lang/blob/0d7a306a128dc4ab731f7d965bc14eefb256eb28/tests/jballerina-unit-test/src/test/resources/test-src/execution/proj1/src/b/main.bal#L1)
Is it required?